docs/getting-started-guides/scratch.md

```diff
@@ -281,12 +281,12 @@ The kubeconfig file for the administrator can be created as follows:
    Guide), you will already have a `$HOME/.kube/config` file.
  - You need to add certs, keys, and the master IP to the kubeconfig file:
     - If using the firewall-only security option, set the apiserver this way:
-      - `kubectl config set-cluster $CLUSTER_NAME --server=http://$MASTER_IP --insecure-skip-tls-verify=true`
+      - `kubectl config set-cluster $CLUSTER_NAME-cluster --server=http://$MASTER_IP --insecure-skip-tls-verify=true`
     - Otherwise, do this to set the apiserver ip, client certs, and user credentials.
-      - `kubectl config set-cluster $CLUSTER_NAME --certificate-authority=$CA_CERT --embed-certs=true --server=https://$MASTER_IP`
-      - `kubectl config set-credentials $CLUSTER_NAME --client-certificate=$CLI_CERT --client-key=$CLI_KEY --embed-certs=true --token=$TOKEN`
+      - `kubectl config set-cluster $CLUSTER_NAME-cluster --embed-certs=true --certificate-authority=$CA_CERT --server=https://$MASTER_IP`
+      - `kubectl config set-credentials $CLUSTER_NAME-admin --embed-certs=true --client-certificate=$CLI_CERT --client-key=$CLI_KEY --token=$TOKEN`
     - Set your cluster as the default cluster to use:
-      - `kubectl config set-context $CLUSTER_NAME --cluster=$CLUSTER_NAME --user=admin`
+      - `kubectl config set-context $CONTEXT --cluster=$CLUSTER_NAME-cluster --user=$CLUSTER_NAME-admin`
```

Comment:
It would probably be less confusing to use `$CLUSTER_NAME` instead of `$CONTEXT`.

Technically, any string will do; we're just defining a key for this cluster/user pair. E.g. for Container Engine we use `gke_$PROJECT_$ZONE_$CLUSTER_NAME` as the context string. `$CLUSTER_NAME` is fine for this guide.